Minor modifications to transpositions of identifications along equivalences

EgbertRijke opened the pull request.

fredrik-bakke commented:
Did you, similarly to me, give up on trying to factor out separate files for transposing identifications along sections and retractions? 😅

EgbertRijke commented:
No, I did not attempt that. Is there any problem with doing that?

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
```diff
@@ -118,27 +162,18 @@ module _
   where
 
-  htpy-map-eq-transpose-equiv :
+  compute-map-eq-tranpsose-equiv :
```
`transpose`*

fredrik-bakke commented:
> No, I did not attempt that. Is there any problem with doing that?

Well, no. It's just a larger job than I think it looks at first glance.

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
````diff
@@ -147,84 +182,70 @@ the original identifications.
+    ( left-inv (is-retraction-map-inv-equiv e _))
+```
+
+### Transposing concatenated identifications along equivalences
+
+Transposing concatenated identifications into a triangle with a transpose of the
+left factor.
+
+```agda
+  triangle-eq-transpose-equiv-concat :
````
Module is carried over to a new section

fredrik-bakke reviewed (approved):
Looks good apart from my few comments!

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
```diff
@@ -86,16 +103,17 @@ module _
-    ( equiv-inv x (map-inv-equiv e y)) ∘e
-    ( eq-transpose-equiv x y) ∘e
-    ( equiv-inv y (map-equiv e x))
+    ( inv-equiv (equiv-ap e _ _)) ∘e
+    ( equiv-concat (is-section-map-inv-equiv e y) _)
```
```suggestion
    ( inv-equiv (equiv-ap e (map-inv-equiv e y) x)) ∘e
    ( equiv-concat (is-section-map-inv-equiv e y) (map-equiv e x))
```

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
```diff
@@ -109,7 +127,33 @@ It is sometimes useful to consider identifications `y ＝ e x` instead of
+  compute-refl-eq-transpose-equiv :
+    {x : A} →
+    map-eq-transpose-equiv e refl ＝ inv (is-retraction-map-inv-equiv e x)
+  compute-refl-eq-transpose-equiv =
+    map-eq-transpose-equiv-inv
+      ( equiv-ap e _ (map-inv-equiv e _))
+      ( ap inv (coherence-map-inv-equiv e _) ∙
```
```suggestion
  compute-refl-eq-transpose-equiv {x} =
    map-eq-transpose-equiv-inv
      ( equiv-ap e x (map-inv-equiv e (map-equiv e x)))
      ( ap inv (coherence-map-inv-equiv e x) ∙
```

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
```diff
@@ -109,7 +127,33 @@ It is sometimes useful to consider identifications `y ＝ e x` instead of
+    map-eq-transpose-equiv-inv e refl ＝ is-retraction-map-inv-equiv e x
+  compute-refl-eq-transpose-equiv-inv {x} =
+    map-eq-transpose-equiv-inv
+      ( equiv-ap e _ _)
+      ( ( right-unit) ∙
+        ( coherence-map-inv-equiv e _))
```
```suggestion
      ( equiv-ap e (map-inv-equiv e (map-equiv e x)) x)
      ( ( right-unit) ∙
        ( coherence-map-inv-equiv e x))
```

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
```diff
@@ -118,27 +162,18 @@ module _
-  htpy-map-eq-transpose-equiv {x} refl =
+  compute-map-eq-tranpsose-equiv {x} refl =
     ( map-eq-transpose-equiv-inv
       ( equiv-ap e x _)
```
```suggestion
      ( equiv-ap e x (map-inv-equiv e (map-equiv e x)))
```

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
```diff
@@ -147,84 +182,70 @@ the original identifications.
-          ( ap-inv (map-equiv e) (map-eq-transpose-equiv-inv e p))))
+    inv
+      ( is-section-map-inv-equiv
+        ( equiv-ap e _ _)
```
```suggestion
        ( equiv-ap e (map-inv-equiv e y) x)
```

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
```diff
@@ -147,84 +182,70 @@ the original identifications.
-          ( ap (map-inv-equiv e) p ∙ is-retraction-map-inv-equiv e x))
-        ( right-unit ∙ triangle-eq-transpose-equiv-inv' p))
+  right-inverse-eq-transpose-equiv {x} refl =
+    ( right-whisker-concat (compute-refl-eq-transpose-equiv e) _) ∙
+    ( left-inv (is-retraction-map-inv-equiv e _))
```
```suggestion
    ( right-whisker-concat
      ( compute-refl-eq-transpose-equiv e)
      ( is-retraction-map-inv-equiv e x)) ∙
    ( left-inv (is-retraction-map-inv-equiv e x))
```

fredrik-bakke commented on src/foundation/transposition-identifications-along-equivalences.lagda.md:
````diff
@@ -147,84 +182,70 @@ the original identifications.
 ```agda
   triangle-eq-transpose-equiv :
````
Module is carried over to new section here too

fredrik-bakke replied:
(Due to your introduction of a new section header)

EgbertRijke commented:
I addressed the comments, except the ones that were about filling in `_` because I actually like this use of underscore and I am much more comfortable with this use of `_` than with the `_` in partially evaluated mixfix operators.
